### PR TITLE
Exchange and credit card terms

### PR DESCRIPTION
**Pull request by SparK-Cruz**
`$anyGateway->exchange()` will give you an instance of Exchange service.
With methods like `localToSite` and `siteToLocal` using gateway's own client connection and configs.

Also, CreditCardGateway can give you a list of possible payment terms (array of `Models\Responses\PaymentTerm`) with info like `instalmentNumber`, `baseAmount` (or sitePrice), `localAmountWithTax`, `tax` and `hasInterests` a boolean flag.

### CODE REVIEW

**Comment by SparK-Cruz** [comment]
2000 LOC for tests 😮

**Comment by SparK-Cruz** [comment]
Build passed but didn't show in github: https://travis-ci.org/ebanx/benjamin/builds/237359212

File: src/Models/Country.php [hunk 1]
```diff
@@ -3,9 +3,20 @@
 
 class Country extends BaseModel
 {
-    const BRAZIL = 'Brazil';
+    const BRAZIL = 'Brasil';
```
Comment (gpressutto5) [inline src/Models/Country.php]: Yeah, the whole lib is in English, I don't think we need to translate Brazil

File: composer.lock [hunk 2]
```diff
@@ -214,12 +214,12 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/doctrine/annotations.git",
-                "reference": "d39134e0c7fbedc6f217f1dfe6ff77cf7fbddbcb"
+                "reference": "7c830a576f9471f71096d200d633b6400513ccfb"
```
Comment (gpressutto5) [inline composer.lock]: Do we need to update composer.lock on this PR?

File: src/Models/Currency.php [hunk 3]
```diff
@@ -34,6 +34,10 @@ public static function globalCurrencies()
 
     public static function localForCountry($country)
```
Comment (gpressutto5) [inline src/Models/Currency.php]: Can we change this method name to `Currency::getLocalByCountry` or something like this?

File: src/Services/Exchange.php [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+     */
+    private $client;
+
+    public function __construct(Config $config, Client $client)
+    {
+        $this->config = $config;
+        $this->client = $client;
+    }
+
+    public function siteToLocal($localCurrency, $localValue = 1)
```
Comment (gpressutto5) [inline src/Services/Exchange.php]: How is the lib user going to use this method?

**Comment by gpressutto5** [comment]
This pull request is huge. Maybe you should have split it into 2 or 3 different PR's. But now that it has been submitted, could you edit the description of this PR with examples of how to use the features you added? I know this is boring, but I've really gotten confused while reviewing.

File: src/Services/Gateways/DebitCard.php [hunk 5]
```diff
@@ -31,7 +31,7 @@ public function create(Payment $payment)
         $adapter = new CardRequestAdapter($payment, $this->config);
         $request = $adapter->transform();
 
-        $body = $this->client->post($request);
```
Comment (SparK-Cruz) [inline src/Services/Gateways/DebitCard.php]: It doesn't fix the tests, it fixes the gateway http call

**Pull request merged**